### ISSUE TITLE
BUG - Highlights Grid stopped working

### ISSUE THREAD

**Issue by ryerman1** [Quicksaver/FindBar-Tweak#240]
Scrollbar Highlights Grid stopped working after update to ver 2.1.7.
Reverting to ver 2.1.5: Highlights Grid works properly.

**Comment by mzso** [comment]
I can confirm this. No highlights on the scrollbar.

(BTW didn't quick findbar used to have the "highlight all", etc buttons? Am I just imagining it?)

**Comment by Quicksaver** [comment]
#239.

> (BTW didn't quick findbar used to have the "highlight all", etc buttons? Am I just imagining it?)

Only if you enable that preference, did you disable it or reset FBT's preferences? (It's also possible the above affected that, I really borked that last update...)

**Comment by mzso** [comment]
You mean  general > findbar mode? I don't know how it got changed. It's been a long while ago when I touched the settings.
That seems to do the trick. (Not that I'm terribly missing it, but that setting doesn't affect the link-only version of the quickfind bar)

**Comment by Quicksaver** [comment]
Hmm, it should. I've already uploaded a fixed version of FBT (so many things were broken I really didn't want to wait on that one). When it updates, if you still don't see the buttons in the links only findbar, please open a new issue and let me know.

**Comment by mzso** [comment]
241
